packages/tasks/src/tasks/chat-completion/spec/input.json

```diff
@@ -17,12 +17,16 @@
 					"content": {
 						"type": "string",
 						"description": "The content of the message."
+					},
+					"name": {
+						"type": "string",
+						"description": "The name of the message."
 					}
 				},
 				"required": ["role", "content"]
 			}
 		},
-		"frequency_penalty": {
+		"presence_penalty": {
```

Comment:
For what I can see [here](https://huggingface.github.io/text-generation-inference/#/Text%20Generation%20Inference/chat_completions), it looks like both `frequency_penalty` and `presence_penalty` exist. What's the difference between those two? Should we add both to the specs?

Reply:
Hi @Wauplin thanks for all of the helpful comments. You're 100% right we should include both of those values.

details: https://platform.openai.com/docs/guides/text-generation/frequency-and-presence-penalties

***screen below for ref
<img width="702" alt="frequency_presence_penalty_diff" src="https://github.com/huggingface/huggingface.js/assets/9896130/c5d4be8c-0fcf-4e3d-a8b0-4029e9090ad4">